ford/__init__.py

```diff
@@ -355,15 +356,30 @@ def main(proj_data,proj_docs,md):
         project.make_links('..')
     else:
         project.make_links(proj_data['project_url'])
+
+    # Define core macros:
+    ford.utils.register_macro('url = {0}'.format(proj_data['project_url']))
+    ford.utils.register_macro( 'media = {0}'.format(
+                                    os.path.join(proj_data['project_url'],
+                                                 'media') ) )
+    ford.utils.register_macro( 'page = {0}'.format(
+                                    os.path.join(proj_data['project_url'],
+                                                 'page') ) )
+    # Register the user defined aliases:
+    for alias in proj_data['alias']:
+        val, key = ford.utils.register_macro(alias)
```

Comment:
Are these used outside of the check on the next line? Could just drop them entirely?

Reply:
Sorry, I fear I don't get the question. What these are you refering to? The alias loop?

Reply:
Sorry, I meant `val, key` -- I just completely didn't type them, and somehow expected you to know what I was thinking!

Reply:
Yes, sorry. Didn't connect the dots. Yes, moving the check into the subroutine should render them useless here. Only the `_MACRO_DICT` should be used elsewhere.